ev-charging/ev-charging.py

```diff
@@ -0,0 +1,140 @@
+import paho.mqtt.publish as publish
```

Comment:
when the button is pressed, you should send also a message for the ev charging sensor. when the charging is ongoing capacity = 0, when the charging is completed (i.e. after the red light stop blinking), capacity = 1.

the service and sensor id are different for the parking and the ev charging sensor

Reply:
additionally it would be good to be able to send a command from the mqtt queue on the ev topic (see the streetlights code) to start and stop the charging.